package.json

```diff
@@ -43,15 +42,12 @@
     "color-convert": "^2.0.1",
     "debounce": "^2.2.0",
     "ical.js": "^2.1.0",
-    "linkify-it": "^5.0.0",
     "markdown-it": "^14.1.0",
     "markdown-it-emoji": "^3.0.0",
     "markdown-it-link-attributes": "^4.0.1",
     "markdown-it-task-lists": "^2.1.1",
     "md5": "^2.3.0",
-    "p-limit": "^6.2.0",
     "sortablejs-vue3": "^1.2.11",
-    "uuid": "^11.0.5",
```

Comment:
:warning: This is a peer-dependency of `@nextcloud/calendar-js@8.0.3`. It has to be removed there first before we can remove it here.

Reply:
PR was merged and released in v8.1.0.